src/main.rs

```diff
@@ -20,10 +20,10 @@ use shakmaty::{fen::Fen, CastlingMode, Chess, FromSetup, Position, Setup};
 
 #[derive(Parser)]
 struct Cli {
-    #[arg(id = "MOVE", default_value = "Qxf7")]
-    mv: String,
+    #[arg(id = "MOVE", default_values = ["Nf6","Qxf7"])]
+    moves: Vec<String>,
 
-    #[arg(default_value = "r1bqkb1r/pppp1ppp/2n2n2/4p2Q/2B1P3/8/PPPP1PPP/RNB1K1NR w KQkq - 4 4")]
+    #[arg(default_value = "r1bqkbnr/pppp1ppp/2n5/4p2Q/2B1P3/8/PPPP1PPP/RNB1K1NR b KQkq - 4 4")]
```

Comment:
You can experiment with matches on https://www.chess.com/analysis?tab=analysis by copying the `initial_state` FEN into the "Setup Position" tab:

![image](https://github.com/nuke-web3/risc0-v1-chess/assets/35669742/184fc9b2-4944-48ac-9bc8-27f67e8aaf5d)

Then `Load` and play the `moves`:

![image](https://github.com/nuke-web3/risc0-v1-chess/assets/35669742/a8db5e1b-c1a7-4288-8350-e956f403255e)